mbedtls/src/pk/mod.rs

```diff
@@ -691,6 +692,13 @@ impl Pk {
                     return Err(Error::RsaOutputTooLarge);
                 }
 
+                // Don't process outside of {2, ..., n-2}
+                let nm1 = self.rsa_public_modulus()?.sub(&Mpi::new(1)?)?;
```

Comment:
So we decided to always do this FIPS check ?

Reply:
Yes, I think it's a sane check we always want. No one should be using this API in any case, they should just use a proper RSA padding.